MatrixSDK/Utils/MXTools.h

```diff
@@ -235,13 +235,13 @@ FOUNDATION_EXPORT NSString *const kMXToolsRegexStringForMatrixGroupIdentifier;
  @discussion
  If the device does not support MP4 file format, the function will use the QuickTime format.
  
- @param videoLocalURL the local path of the video to convert.
+ @param videoAsset an AVAsset for the video to convert.
  @param targetFileSize a file size limit to aim for.
  @param success A block object called when the operation succeeded. It returns
  the path of the output video with some metadata.
  @param failure A block object called when the operation failed.
  */
-+ (void)convertVideoToMP4:(NSURL*)videoLocalURL
```

Comment:
Can't you add a new method with a `AVAsset` as you did for `MXRoom` instead of changing the prototype of this one?